packages/upgrader/src/index.ts

```diff
@@ -198,8 +198,6 @@ function* triggerControllerDeploymentUpgrade() {
     // handle upgrades from clusters that are not running the cortex-operator
     yield call(cortexOperatorPreamble, kubeConfig);
 
-    yield call(upgradeControllerConfigMap, kubeConfig);
```

Comment:
@sreis was this call added for a reason in https://github.com/opstrace/opstrace/pull/869/files#diff-d85ea48a17112e49ec6714d9c57517cc0192ca603adad37fbb65bc893c52db0bR201? It looks extraneous to me but I can't be sure.

Reply:
No, it's a good catch from you to delete one of the calls.